include/ck_tile/ops/gemm/kernel/gemm_tile_partitioner.hpp

```diff
@@ -62,12 +62,10 @@ struct GemmTile1DPartitioner
         return integer_divide_ceil(K, KPerBlock);
     }
 
-    CK_TILE_DEVICE auto operator()(index_t blockOffset, index_t NBlockSize)
```

Comment:
Right now the function in https://github.com/ROCm/composable_kernel/pull/1756/files#diff-1452d09e42d9aed38e087f552e940394aae58d72f2991fa842fa15caced37854R55 is not used anywhere.